### DISCUSSION TITLE
py文件运行后台问题

### DISCUSSION THREAD

**Discussion by zouyahui** [GaiZhenbiao/ChuanhuChatGPT#363] [Q&A] [answered]
py文件运行完弹出链接，点击后浏览器中开始聊天，这时python程序可以终止吗？

**Comment by GaiZhenbiao** [comment, marked as answer]
不可以。你可以让服务器在后台运行。例如在Linux/macOS中，可以使用nohup命令。

**Reply by zouyahui** [reply]
谢谢，那这样就不能同时调试其他python代码了吧（让川虎chatgpt改代码调试）

**Comment by GaiZhenbiao** [comment]
当然可以了。没啥不可以的。不冲突的

**Reply by zouyahui** [reply]
pycharm可以同时运行两个python代码吗，刚才在网上搜了一下  ，尝试都不行，是pycharm版本的问题吗

**Comment by GaiZhenbiao** [comment]
1. 我不使用Pycharm。
2. 按常理，是可以的。

**Reply by zouyahui** [reply]
好像可以了，谢谢~
